src/Pagination.jsx

```diff
@@ -190,6 +190,12 @@ class Pagination extends React.Component {
     return this.state.current < this._calcPage();
   }
 
+  _runIfEnter(event, callback) {
+    if (event.keyCode === 'Enter') {
```

Comment:
You should use `event .key === 'Enter'` or `event.charCode  === 13`, see https://facebook.github.io/react/docs/events.html#keyboard-events